arangod/MMFiles/MMFilesRestReplicationHandler.h

```diff
@@ -38,6 +38,16 @@ class MMFilesRestReplicationHandler : public RestReplicationHandler {
 
  public:
   RequestLane lane() const override final {
```

Comment:
why is that implementation not in the superclass ?

Reply:
good point, original was not as well, fixing it.